readme.md

```diff
@@ -105,34 +105,34 @@
 	- [Functional Programming](https://github.com/stoeffel/awesome-fp-js)
 	- [Observables](https://github.com/sindresorhus/awesome-observables)
 	- [npm scripts](https://github.com/RyanZim/awesome-npm-scripts) - Task runner.
-- [Swift](https://github.com/matteocrippa/awesome-swift)
+- [Swift](https://github.com/matteocrippa/awesome-swift)-  a general-purpose, multi-paradigm, compiled programming language developed by Apple Inc.
 	- [Education](https://github.com/hsavit1/Awesome-Swift-Education)
 	- [Playgrounds](https://github.com/uraimo/Awesome-Swift-Playgrounds)
-- [Python](https://github.com/vinta/awesome-python)
+- [Python](https://github.com/vinta/awesome-python) - a widely used high-level programming language for general-purpose programming, created by Guido van Rossum.
```

Comment:
Too generic again, and if you don't know Python, you don't know `Guido van Rossum`, so that last part is moot.